### PR TITLE
Support for build-args of docker build

### PR DESCRIPTION
**Pull request by alenkacz**
In 1.9 there is new property - build-args that can be specified for
docker build command

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java [hunk 1]
```diff
@@ -87,6 +88,9 @@
     @CheckForNull
     public String getCpusetcpus();
 
+    @CheckForNull
+    public Map<String, String> getBuildArgs();
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java]: since what API  version?

Comment (alenkacz) [inline src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java]: 1.9 https://github.com/docker/docker/releases/tag/v1.9.0

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java]: That's docker version, not docker API https://docs.docker.com/engine/reference/api/docker_remote_api/

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java]: Looks like `1.21`

Comment (alenkacz) [inline src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java]: Yep. 1.21

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java]: Could you add javadoc? Example:
https://github.com/docker-java/docker-java/blob/9f779727004c4f87e2b4426145e3b1bdb302b39f/src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java#L360-L364
reference to field
https://github.com/docker-java/docker-java/blob/9f779727004c4f87e2b4426145e3b1bdb302b39f/src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java#L278-L281

**Comment by alenkacz** [comment]
I have added javadoc with api version information

**Comment by KostyaSha** [comment]
LGTM, @marcuslinke ?

**Comment by marcuslinke** [comment]
@alenkacz @KostyaSha I'm OK with it but it needs to be implemented/tested for the netty based `BuildImageCmdExec` also before this PR can be merged.

**Comment by alenkacz** [comment]
I have added that netty based implementation. By the way - keeping two implementations like this is not a good architectural decision - I think. It is really error prone...

Also I think that you should detach this forked repository if you are not planning on merging it back to the previous one. Because if it is like this, contributions are not counted for example - because they are counted only after fork is merged to the original repository. Which will never happen.

How to detach fork is written here - https://www.quora.com/Git-revision-control/How-can-one-detach-a-forked-project-in-GitHub I used GitHub support in the past and they are really quick.

**Comment by marcuslinke** [comment]
@alenkacz Thanks! The netty implementation will supersede the old jersey implementation in the near future so this is a temporary situtation only that will stay until after the next release ( > v3.0.0). 

I will try to reach Github support to detach the repository as you suggested. Didn't know this is possible.

Will merge this ASAP. Thanks for contributing!

**Comment by KostyaSha** [comment]
> Also I think that you should detach this forked repository if you are not planning on merging it back to the previous one. Because if it is like this, contributions are not counted for example - because they are counted only after fork is merged to the original repository. Which will never happen.

Fork shows history, initial repository is not used anymore and technically i think it in mergeable state.

**Comment by KostyaSha** [comment]
@marcuslinke does this fork changed artifactId? If not then i see no reasons for breaking fork chain.

**Comment by marcuslinke** [comment]
@KostyaSha The groupId was changed from `com.kpelykh` to `com.github.docker-java`. However, beside this I see one main reason to detach: When you create a pull request the default base fork is set to https://github.com/kpelykh/docker-java which annoys me everytime. So do you have any objections against detaching?

**Comment by KostyaSha** [comment]
Let's try.

**Comment by marcuslinke** [comment]
Done.

**Pull request merged**